aiosmtplib/errors.py

```diff
@@ -78,6 +79,16 @@ def __init__(self, code: int, message: str) -> None:
         self.args = (code, message)
 
 
+class SMTPConnectResponseError(SMTPResponseException, SMTPConnectError):
+    """
+    An error occurred while connecting to the SMTP server.
+    """
+
+    def __init__(self, code: int, message: str) -> None:
```

Comment:
I think we don't need this method, if you just leave it out SMTPResponseException's init will be used.

Reply:
Done!